classes/local/backup/restore_lifecycle_workflow.php

```diff
@@ -102,6 +101,7 @@ private function load_workflow() {
         $this->workflow->timeactive = null;
         $this->workflow->timedeactive = null;
         $this->workflow->sortindex = null;
+        $this->workflow->id = workflow_manager::insert_or_update($this->workflow);
```

Comment:
This line, and returning the workflow id in `insert_or_update` is theoretically not necessary, because $this->workflow is passed by reference to it, right?

Reply:
Could be